lib/calendar.ts

```diff
@@ -334,7 +334,7 @@ export class Calendar implements Temporal.Calendar {
   static from(item: Params['from'][0]): Return['from'] {
     return ES.ToTemporalCalendar(item);
   }
-  [Symbol.toStringTag]!: 'Temporal.Calendar';
```

Comment:
Does the reason we originally added this assertion (see https://github.com/js-temporal/temporal-polyfill/pull/53#issuecomment-924524613) not apply anymore? Does TS now emit `[Symbol.toStringTag]` even without the `!` assertion?

Reply:
Note that this is not actually a non-null assertion, it's a "[definite assignment assertion](https://www.typescriptlang.org/docs/handbook/2/classes.html)" that means that this field will be initialized by non-constructor code.